packages/amplify_datastore/ios/Classes/GetJsonValue.swift

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import Amplify
+
+func getJSONValue(_ jsonDict: [String: Any]) -> [String: JSONValue]{
+    guard let jsonData = try? JSONSerialization.data(withJSONObject: jsonDict) else {
+        print("JSON error")
+        return [:]
```

Comment:
If I'm reading this right, it says:

> If there's a serialization failure, just ignore it, and return an empty map.

Potentially, this is just pushing errors down the road, where there will be more debugging steps needed, to determine their root cause.

Would it not be better to "fail fast" on bad JSON?

Reply:
Agree with this, I am throwing an error in the save PR around being unable to convert to JSON.

Reply:
Good point- I'll do this.

Reply:
Done.